blessings/tests.py

```diff
@@ -10,13 +10,14 @@
 
 """
 from __future__ import with_statement  # Make 2.5-compatible
+from __future__ import absolute_import
```

Comment:
Is this (and the one in the other file) just for hygiene? I don't see that it changes any behavior in the context of this project.

Reply:
Also, it doesn't do anything in Python 2.7 and later, which is all we support now. :-) https://www.python.org/dev/peps/pep-0328/